sigstore/_internal/tuf.py

```diff
@@ -66,6 +74,21 @@ def _get_dirs(url: str) -> tuple[Path, Path]:
     return (tuf_data_dir / repo_base), (tuf_cache_dir / repo_base)
 
 
+def _is_timerange_valid(period: TimeRange | None, allow_expired: bool) -> bool:
```

Comment:
Extreme nitpick, forces all users of this function to specify the `allow_expired` kwarg explicitly.

```suggestion
def _is_timerange_valid(period: TimeRange | None, *, allow_expired: bool) -> bool:
```